pycloudlib/ec2/instance.py

```diff
@@ -126,6 +126,12 @@ def delete(self, wait=True):
         if wait:
             self.wait_for_delete()
 
+    def enable_ipv6_instance_metadata(self):
```

Comment:
I don't think we want to typically run this after the instance launches because there could be a race with the instance coming up earlier than our IPv6 IMDS metadata makes it through AWS backplane, so cloud-init may have already passed the point where early IMDS detection passed on an initially IPv4-only IMDS.

I think we actually want this at launch which looks like we can provide via something like [create_instances kwargs MetadataOption={'HttpProtocolIpv6': 'enabled'}](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances). 
I'm testing this suggestion now. That said, we might want an easy way to provide a simple param during ec2 instance launch to specify IMDS 'ipv4'|'ipv6'|'dual-stack' at instance launch time so we don't have to digup and remember these options at every call-site.